app/controllers/application_controller/authentication_dependency.rb

```diff
@@ -48,6 +51,11 @@ def logged_in?
     !current_user.nil?
   end
 
+  def log_out
```

Comment:
Moved log_out method to authentication_dependency so it can be used here and in sessions controller